src/routes/feedback/components/__tests__/__snapshots__/FeedbackLink.spec.js.snap

```diff
@@ -0,0 +1,23 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`FeedbackLink should render a negative FeedbackLink 1`] = `
+<withProps(Connect(Link))
+  to="undefined?feedback=down"
```

Comment:
I think pathname shouldn't be undefined

Reply:
Testing will be done in a separate PR.

Reply:
This is just a leftover from WEBAPP-311.